README.md

```diff
@@ -9,3 +9,7 @@ My hope is that this will scratch 2 of my current itches:
 
 I have some basic ideas for features I'd like to see in this. If you have any, I'd love to hear them - just create a new issue :).  
 
+## Developer Setup
+### Google Authentication
+The [Ueberauth documentation](https://hexdocs.pm/ueberauth_google/extra-readme.html) is very helpful,
+but the Google side of things is a bit difficult to follow. When you follow their link to the Google Developer Console, you need to create an account that is linked to a credit card. Next, you need to enable the Google+ API. From there, you need to configure the credentials. Set up the OAuth consent screen and generate OAuth credentials for your development environment. Add `http://localhost:4000/auth/google/callback` as the authorized callback URL. Lastly, you should the credentials and test.  
```

Comment:
@matt-keibler wasn't quite sure what that last sentence meant but it made me chuckle so we're leaving it ;) lol